screenly-cast/inc/assets/js/scripts.js

```diff
@@ -0,0 +1,97 @@
+jQuery(document).ready(function($) {
+	
+	$('#srlySelectImage').click(function(e) {
+		e.preventDefault();
+
+		var custom_uploader = wp.media({
+			title: 'Select Logo',
+			button: {
+				text: 'Select as Logo'
+			},
+			multiple: false  // Set this to true to allow multiple files to be selected
+		})
+		.on('select', function() {
+			var attachment = custom_uploader.state().get('selection').first().toJSON();
+			$('#media_image').attr('src', attachment.url);
+			$('#screenly_options_logo').val(attachment.url);
+		})
+		.open();
+	});
+	$('#media_image').attr('src', $('#screenly_options_logo').val());
+	$('.srly_color_picker').wpColorPicker();
+	
+	$('#srly_brand_logo_width').change(function(){
```

Comment:
Is it possible to use default WP's UI controls in settings's page? It's not userfriendly, especially on mobile and tablets